src/main/java/org/jabref/gui/preferences/appearance/AppearanceTabViewModel.java

```diff
@@ -82,54 +75,34 @@ public void setValues() {
         fontSizeProperty.setValue(String.valueOf(appearancePreferences.getMainFontSize()));
 
         Theme currentTheme = appearancePreferences.getTheme();
-        if (currentTheme.getType() == Theme.Type.LIGHT) {
+        if (currentTheme.getType() == Theme.Type.DEFAULT) {
             themeLightProperty.setValue(true);
             themeDarkProperty.setValue(false);
             themeCustomProperty.setValue(false);
-        } else if (currentTheme.getType() == Theme.Type.DARK) {
+        } else if (currentTheme.getType() == Theme.Type.EMBEDDED) {
```

Comment:
Default and embedded are both embedded, I would name it Embedded_Light and embedded_dark

Reply:
Light is not really a theme, it's rather the lack of existence of a theme.
EMBEDDED is to prepare more embedded themes (see the constructor of Theme), just by providing the filename as a static variable like "dark.css".